### PR TITLE
Update Learn-Back-End-Dev.md

### PR DESCRIPTION
**Pull request by troynunnally**

### CODE REVIEW

File: Learn-Back-End-Dev.md [hunk 1]
```diff
@@ -5,4 +5,7 @@
 
 ## Ruby
 * [Rails Testing](https://github.com/eliotsykes/rspec-rails-examples)
-* [React on Rails](https://github.com/shakacode/react_on_rails)
\ No newline at end of file
+* [React on Rails](https://github.com/shakacode/react_on_rails)
+
+## NodeJS
+* [Authenticating A User using JSON Web Tokens](http://jasonwatmore.com/post/2016/08/16/angular-2-jwt-authentication-example-tutorial)
```
Comment (x6iae) [inline Learn-Back-End-Dev.md:11]: Is this not more related to AngularJS than NodeJS?

Comment (troynunnally) [reply Learn-Back-End-Dev.md:11]: Yes, you are correct. Please excuse mistake. I will make new PR shortly.
